openid-sse-framework-1_0.txt

```diff
@@ -1448,47 +1397,211 @@ Tulshibagwale, et al.   Expires December 10, 2021              [Page 25]
-   |      |                                                            |
-   | 404  | if there is no Event Stream configured for this Event      |
-   |      | Receiver                                                   |
-   +------+------------------------------------------------------------+
-
-                Table 4: Update Stream Configuration Errors
-
-7.1.2.3.  Removing a Stream Configuration
-
-   An Event Receiver removes the configuration of a stream by making an
-   HTTP DELETE request to the Configuration Endpoint.  On receiving a
-   request the Event Transmitter responds with a "200 OK" response if
-   the configuration was successfully removed.
-
-   The following is a non-normative example request to remove an Event
-   Stream's configuration:
-
-   DELETE /sse/stream HTTP/1.1
+        +======+==================================================+
+        | Code | Description                                      |
+        +======+==================================================+
+        | 202  | if the replace request has been accepted, but    |
+        |      | not processed.  Receiver MAY try the same        |
+        |      | request later in order to get processing result. |
+        +------+--------------------------------------------------+
+        | 400  | if the request body cannot be parsed, a read-    |
+        |      | only property is incorrect, or if the request is |
+        |      | otherwise invalid                                |
+        +------+--------------------------------------------------+
+        | 401  | if authorization failed or it is missing         |
+        +------+--------------------------------------------------+
+        | 403  | if the Event Receiver is not allowed to replace  |
+        |      | the stream configuration                         |
+        +------+--------------------------------------------------+
+        | 404  | if there is no Event Stream with the given       |
+        |      | "stream_id" for this Event Receiver              |
+        +------+--------------------------------------------------+
+
+                Table 4: Replace Stream Configuration Errors
+
+7.1.1.5.  Deleting a Stream
+
+   An Event Receiver deletes a stream by making an HTTP DELETE request
+   to the Configuration Endpoint.  On receiving a request the Event
+   Transmitter responds with a 201 OK response if the configuration was
+   successfully removed.
+
+   The DELETE request MUST include the "stream_id" as a parameter in
+   order to identify the correct Event Stream.
+
+   The following is a non-normative example request to delete an Event
+   Stream:
+
+  DELETE /sse/stream?stream_id=f67e39a0a4d34d56b3aa1bc4cff0069f HTTP/1.1
+  Host: transmitter.example.com
+  Authorization: Bearer eyJ0b2tlbiI6ImV4YW1wbGUifQo=
+
+                Figure 22: Example: Delete Stream Request
+
+   Errors are signaled with HTTP status codes as follows:
```

Comment:
There is no description of what the success response is to a deletion request. Please add.

Reply:
It is specified in the paragraphs above the normative examples, like all of the other endpoints. Although I should have made it a `204` instead of a `201`. Fixing that now.